### PR TITLE
Bump to 3.1.0 and add myself as maintainer

### PR DESCRIPTION
**Pull request by tschoonj**
@ocefpaf
@marcelotrevisani
@isuruf

This feedstock appears to be unmaintained and I would like to help out here.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,23 +16,22 @@ source:
...
 
   run:
-    - python
+    - python >=3.5
     - pbr !=2.1.0,>=2.0.0
-    - cmd2 !=0.8.3,<0.9.0  # [py2k]
-    - cmd2 !=0.8.3  # [py3k]
-    - prettytable <0.8,>=0.7.2
+    - cmd2 !=0.8.3
+    - prettytable >=0.7.2
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:31]:
```suggestion
    - prettytable >=0.7.2,<0.8
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -16,23 +16,22 @@ source:
...
     - pbr !=2.1.0,>=2.0.0
 
   run:
-    - python
+    - python >=3.5
     - pbr !=2.1.0,>=2.0.0
-    - cmd2 !=0.8.3,<0.9.0  # [py2k]
-    - cmd2 !=0.8.3  # [py3k]
-    - prettytable <0.8,>=0.7.2
+    - cmd2 !=0.8.3
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:30]:
```suggestion
    - cmd2 !=0.8.3,<0.9.0,>=0.8.0
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -16,23 +16,22 @@ source:
 build:
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:18]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -16,23 +16,22 @@ source:
 build:
   number: 0
```
Comment (marcelotrevisani) [inline recipe/meta.yaml:17]:
```suggestion
  number: 1
```

File: recipe/meta.yaml (hunk 1 above)
Comment (tschoonj) [reply recipe/meta.yaml:31]: Same remark as in https://github.com/conda-forge/staged-recipes/pull/11675#discussion_r427752664

I am not convinced that these upper constraints are justified.

Comment (marcelotrevisani) [reply recipe/meta.yaml:31]: Just following the metadata provided by the `cliff`` package

Comment (tschoonj) [reply recipe/meta.yaml:31]: May I suggest that for now we remove the upper constraints: if there is a problem running these packages against the latest version of the dependencies, I will gladly reinstate the constraints.

**Pull request merged**